test cases/common/58 run target/fakeburner.py

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
+
+from __future__ import print_function
 
```

Comment:
But this script will always be run with Python 3 if we use the module, and in that case don't we want to know if it's not run with Python 3 for some reason?